### DISCUSSION TITLE
Want to contribute

### DISCUSSION THREAD

**Discussion by CromixPT** [maeek/ha-aux-cloud#33] [General]
Hi there,

Are you still actively developing this integration?
Im looking into assist with this

**Comment by maeek** [comment]
Hello, I'm not super familiar with developing Home Assistant integrations and don't have much time lately, hence the current stall in my project progress, but feel free to start working on the integration and post your questions here. I can check in every now and then to provide answers and guidance when needed about how the aux cloud API works as I spent some time researching it.

**Comment by CromixPT** [comment]
Do you have any documentation or insights you can share on aux cloud API so that I can make head way on it?

**Comment by maeek** [comment]
I've added jupyter notebook with some info on how the API and a wrapper for it that I created works. You can find it here https://github.com/maeek/ha-aux-cloud/blob/main/custom_components/aux-cloud/api/aux-usage.ipynb

**Comment by maczet** [comment]
Can you get me access to branch creating. I will try to contribute. 
I not familiar with HA component too.

**Comment by maczet** [comment]
How can I contribute? I should create feature branch and marge to develop?
I can not create any new branch.

**Comment by maeek** [comment]
The best way to contribute, is to fork this repository. Once you fork it you can make any changes you want in your copy and then create a pull request to this repo. Here's github docs that will walk you through the process https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

**Comment by CromixPT** [comment]
@maeek have you removed the juniper notebook?

**Comment by maeek** [comment]
Hi, nope. It's in root folder https://github.com/maeek/ha-aux-cloud/blob/main/aux_cloud_api_usage.ipynb

**Comment by maczet** [comment]
I think this one topic can be move to discussions.

**Comment by libtiff** [comment]
what tools did you use to decrypt HTTPS traffic from APK?

Im trying to contribute and resolve the "Error fetching device params for 00000000000000000000e81656cdae44" with my AUX heatpump
